docs/samples/llmisvc/opt-125m/llm-inference-service-pd-facebook-opt-125m-cpu.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   template:
     initContainers:
       - name: llm-d-routing-sidecar
-        image: quay.io/pierdipi/llm-d-routing-sidecar:24c4316aaa
+        image: ghcr.io/llm-d/llm-d-routing-sidecar:v0.2.0
```

Comment:
can we just drop it?

Reply:
Done in [63e570](https://github.com/opendatahub-io/kserve/pull/783/commits/e63e5704c7659771132b6e0a06293fa5413dc3c4)